service/src/test/java/com/kodemon/service/test/TimeServiceTest.java

```diff
@@ -28,7 +29,10 @@
 
     @Test
     public void currentDateTest() {
-        assertThat(timeService.currentDate(), is(equalTo(new Date())));
+        long myDate = new Date().getTime();
+        long serviceDate = timeService.currentDate().getTime();
+        assertThat(myDate-5L, is(lessThan(serviceDate)));
```

Comment:
my bad.
Please do `Math.abs(myDate - serviceDate) < 5L`, but extract 5L to the static final constant pls.